common/src/main/java/com/wynntils/models/players/GuildModel.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.players;
+
+import com.wynntils.core.components.Model;
+import java.util.List;
+
+public class GuildModel extends Model {
+
+    public GuildModel() {
+        super(List.of());
+    }
+
+    public enum GuildRank {
```

Comment:
Put this in `com.wynntils.models.players.type` and then there is no need for a new model